frontend/src/constants/API.js

```diff
@@ -40,6 +40,9 @@ export const MINE_MANAGER_HISTORY = (mineNo) =>
 export const MINESPACE_USER = "/users/minespace";
 export const PROVINCE_CODES = "/parties/sub-division-codes";
 
+export const MINE_VERIFIED_STATUSES = (params) =>
+  params ? `/mines/verified-status?${queryString.stringify(params)}` : "/mines/verified-status";
```

Comment:
I'm in favour of setting the default `params` value to an empty object (as I've done on line 16). Then there's no need for the ternary. It also handles the `null` check, so the function will never blow up so long as it either receives `null`, `undefined` or an object.

Reply:
ok, changed.